readme.MD

```diff
@@ -59,9 +59,13 @@ If you want to only test a specific area of a page, or wish to exclude some part
 
 Axe-cli will look for locally available versions of axe-core. If the directory from where you start axe-cli has an `axe.js` file, or has a `node_modules` directory with axe-core installed in it. Axe-cli will use this version of axe-core instead of the default version installed globally.
 
+To specify the exact file axe-core file axe-cli should use, you can use the `--axe-source` flag (`-a` for short), with a relative or absolute path to the file.
```

Comment:
In the future we should put stuff like this into a separate PR to keep things focused.